streampark-flink/streampark-flink-kubernetes/pom.xml

```diff
@@ -96,6 +96,24 @@
             <version>${snakeyaml.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-core</artifactId>
+            <optional>true</optional>
+        </dependency>
```

Comment:
Are all dependencies necessary?

In your kubernetes env, you use hdfs as your archive storage, so you add these dependencies, right?

If yes.  If some users use s3, it will miss some jar, right?

Reply:
For different storage media, to join different related dependencies, here is the limit range: hdfs, s3 . Or is there a better way